GCC warning old-style-declaration on bluetooth host/att.c

**Describe the bug**
warning old-style-declaration on bluetooth host/att.c:137
`/zephyr/subsys/bluetooth/host/att.c:137:1: error: 'static' is not at beginning of declaration [-Werror=old-style-declaration]`

**To Reproduce**
Use arm-zephyr-eabi-gcc (Zephyr SDK 0.17.0) 12.2.0

**Expected behavior**
No warnings

**Impact**
I have to activate for my project : zephyr_compile_options("-Wno-old-style-declaration")

--- a/subsys/bluetooth/host/att.c
+++ b/subsys/bluetooth/host/att.c
@@ -133,7 +133,7 @@ static uint16_t bt_att_mtu(struct bt_att_chan *chan)
 /* Descriptor of application-specific authorization callbacks that are used
  * with the CONFIG_BT_GATT_AUTHORIZATION_CUSTOM Kconfig enabled.
  */
-const static struct bt_gatt_authorization_cb *authorization_cb;
+static const struct bt_gatt_authorization_cb *authorization_cb;
 
 /* ATT connection specific data */
 struct bt_att {

